packages/bpk-tokens/src/android/base/typography.json

```diff
@@ -13,16 +11,30 @@
       "type": "font",
       "category": "typesettings"
     },
+
+    "FONT_SIZE_CAPS": {
+      "value": "{!FONT_SIZE_CAPS}",
+      "type": "font-size",
+      "category": "typesettings"
+    },
     "FONT_SIZE_XS": {
       "value": "{!FONT_SIZE_XS}",
       "type": "font-size",
-      "category": "typesettings"
+      "category": "typesettings",
+      "meta": {
+        "deprecated": true
+      }
     },
     "FONT_SIZE_SM": {
       "value": "{!FONT_SIZE_SM}",
       "type": "font-size",
       "category": "typesettings"
     },
+    "FONT_SIZE_MD": {
```

Comment:
I thought we agreed to map these to the old styles and adding `caps` and `xxxl`. i.e.

+ _ -> caps **New**
+ s -> xs
+ m -> s
+ base -> base
+ l -> l
+ xl -> xl
+ xxl -> xxl
+ _ -> xxxl **new**

Reply:
I am not expeciting `MD` to be introduced here

Reply:
I remember something like this coming up in discussions. So was the consensus that we should overrule Will's naming and provide consumers with xs,s,base,lg,xl,xxl,etc or did we decide that this mapping should exist internally so that consumers have the new names exposed to them?

Reply:
We decided where possible we will map to the old names and only introduce CAPS and xxxl as new sizes. cc @willhg

Reply:
Cool 👍

Reply:
spoken to @willhg about this and he's happy to crack on with caps, xs, s, base, ...